packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -42,7 +42,7 @@ test.describe('PieCookieBanner - Component tests', () => {
         expect(cookieBanner).toBeVisible();
     });
 
-    test('clicking "Accept all" emits the correct event and closes the cookie banner', async ({ mount, page }) => {
+    test('should clicking "Accept all" emits the correct event and closes the cookie banner', async ({ mount, page }) => {
```

Comment:
I think these sentences need fixing as it doesn't read right at the moment.

Reply:
something like `should emit the correct event and close the cookie banner when "Accept all" is clicked`

Reply:
lol sorry thats awful isn't it - let me re-word